scripts/gis-material-update/validate-deploy/modules/hsl.py

```diff
@@ -0,0 +1,19 @@
+import logging
+
+from modules.config import Config
+from modules.common import *
+from modules.gis_validate_deploy import GisProcessor
+
+class HslBuses(GisProcessor):
+    """Process HSL bus lines."""
+
+    def __init__(self, cfg: Config):
+        self._module = "hsl"
+        self._filename = cfg.target_buffer_file(self._module)
+        GisProcessor.__init__(self, cfg)
+
+    def get_temp_data(self, cfg: Config):
+        GisProcessor.get_temp_data(self, cfg)
+
+    def validate_deploy(self):
+        GisProcessor.validate_deploy(self)
```

Comment:
If the GisProcessor methods are not marked as `@abstractmethod`, the subclasses don't have to implement them, unless they need a different implementation. They inherit the parent class's methods directly.

Reply:
Removed methods. Same done also other modules except autoliikennemaarat.